charts/woodpecker/README.md.gotmpl

```diff
@@ -61,20 +45,7 @@ resource "helm_release" "woodpecker" {
 
 <summary>To 1.0.0</summary>
 
-* split `metrics.enabled` and the PodMonitor deployment to `prometheus.podmonitor.enabled`
-* move `metrics.interval` to `prometheus.podmonitor.interval`
+- split `metrics.enabled` and the PodMonitor deployment to `prometheus.podmonitor.enabled`
+- move `metrics.interval` to `prometheus.podmonitor.interval`
 
 </details>
-
-## Versioning
```

Comment:
Shouldn't this part be moved to the top level Readme?

Reply:
I don't think so, see https://github.com/woodpecker-ci/helm/pull/101/files#r1392551144